torch/cuda/streams.py

```diff
@@ -75,13 +75,7 @@ def query(self):
         Returns:
             A boolean indicating if all kernels in this stream are completed.
         """
-        with torch.cuda.device(self.device):
-            res = cudart().cudaStreamQuery(self)
-            if res == cudaStatus.ERROR_NOT_READY:
```

Comment:
Do we already have an error checking implementation in C++ that I can call?  Or should I just return `cudaError_t` and keep the error checking in Python?